unconference/Project_taxonomy_recommendations.md

```diff
@@ -0,0 +1,577 @@
+# Group 3: Project taxonomy and recommendations for Diversity Equity and Inclusion
+
+
+## Working Group Members:
```

Comment:
see the top of https://github.com/numfocus/disc-unconference-2023-projects/blob/main/unconference/mentoring.md for some info we should add that would include the author info below